src/test/java/seedu/address/storage/JsonSerializableEventBookTest.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.storage;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.EventBook;
+import seedu.address.model.event.Event;
+import seedu.address.model.event.EventName;
+import seedu.address.model.person.UniquePersonList;
+import seedu.address.testutil.TypicalPersons;
+
+public class JsonSerializableEventBookTest {
+
+    @Test
+    public void toModelType_validEventDetails_returnsEventBook() throws IllegalValueException {
```

Comment:
toModelType_validEventDetails_success() would be better. should make it consistent with the JsonSerializableAddressBookTest.java format